deployment/environment/web/sdk/client/client.go

```diff
@@ -32,7 +32,7 @@ type Client interface {
 	DeleteJobDistributorChainConfig(ctx context.Context, id string) error
 	GetJobProposal(ctx context.Context, id string) (*generated.GetJobProposalJobProposal, error)
 	ApproveJobProposalSpec(ctx context.Context, id string, force bool) (*JobProposalApprovalSuccessSpec, error)
-	CancelJobProposalSpec(ctx context.Context, id string) (*generated.CancelJobProposalSpecResponse, error)
+	CancelJobProposalSpec(ctx context.Context, id string) (*generated.CancelJobProposalSpecCancelJobProposalSpecCancelJobProposalSpecSuccessSpecJobProposalSpec, error)
```

Comment:
Is this naming expected? Looks like `CancelJobProposalSpec` is repeated a few times.

Reply:
This is auto generated I have not changed anything